lib/network/modules/components/nodes/shapes/CustomShape.js

```diff
@@ -1,6 +1,6 @@
-'use strict';
+"use strict";
 
-import ShapeBase from '../util/ShapeBase'
+import ShapeBase from "../util/ShapeBase";
```

Comment:
Could you remove the reformatting (it makes git blame much nicer to walk through), please? We can reformat the codebase with Prettier and force consistent formatting, which is probably a very good idea, but this should be done in a separate PR.